includes/amp-helper-functions.php

```diff
@@ -653,12 +653,14 @@ function amp_get_current_url() {
 	if ( isset( $parsed_url['port'] ) ) {
 		$current_url .= ':' . $parsed_url['port'];
 	}
-	$current_url .= '/';
+	
+	$current_url .= '/' . trim($parsed_url['path'], '/') . '/' ;
```

Comment:
For a site that is located at the root, wouldn't this result in `//` being appended?